lms/static/js/financial-assistance/views/financial_assistance_form_view.js

```diff
@@ -101,6 +103,28 @@
 
                 setExtraData: function(data) {
                     return _.extend(data, this.user_details);
+                },
+
+                validateCountry: function() {
+                    var $submissionContainer = $('.submission-error'),
+                        $errorMessageContainer = $submissionContainer.find('.message-copy'),
+                        $countryLabel = $('#user-country-title'),
+                        errorMsg = [
+                            "Please go to your <a href=\"",
+                            this.context.account_settings_url,
+                            "\">profile page</a> and provide your country of residence."
+                        ];
+
+                    if( !this.model.get('country') ){
+                        $countryLabel.addClass('error');
+                        $errorMessageContainer.append(
+                            "<li>" +
+                            gettext(errorMsg.join('')) +
```

Comment:
This won't work; `gettext` needs to have literal strings passed to it to mark for translation (see [our internationalization guidelines](https://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html)).

You probably want something like

```javascript
interpolate(
    gettext('Please go to your %(link_start)profile page%(link_end) and provide your country of residence.'),
    {link_start: '<a href="' + this.context.account_settings_url + '">', link_end: '</a>'}
)
```